template/helper/player.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @package    DOCman
+ * @copyright   Copyright (C) 2011 - 2014 Timble CVBA (http://www.timble.net)
+ * @license     GNU GPLv3 <http://www.gnu.org/licenses/gpl.html>
+ * @link        http://www.joomlatools.com
+ */
+
+class ComFilesTemplateHelperPlayer extends KTemplateHelperAbstract
+{
+    protected static $HTML5_VIDEO_FORMATS = array('mp4', 'webm', 'ogg');
```

Comment:
@rmdstudio I would rename this to VIDEO_FORMATS, this is not related to HTML5 per se.